lambda/src/main/resources/db/migration/V8__add-matches-table

```diff
@@ -0,0 +1,38 @@
+DROP TABLE "FFIDMetadata";
+CREATE TABLE "FFIDMetadata"
+(
+    "FFIDMetadataId" uuid NOT NULL,
+    "FileId" uuid NOT NULL,
+    "Software" text COLLATE pg_catalog."default" NOT NULL,
+    "SoftwareVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "Datetime" timestamp with time zone not null,
+    "BinarySignatureFileVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "ContainerSignatureFileVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "Method" text COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT "FFIDMetadata_pkey" PRIMARY KEY ("FFIDMetadataId"),
+    CONSTRAINT "FFIDMetadata_Consignment_fkey" FOREIGN KEY ("FileId")
+        REFERENCES "File" ("FileId") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+)
+WITH (
+    OIDS = FALSE
+);
+
+CREATE TABLE "FFIDMetadataMatches"
```

Comment:
Today I learned that it's possible to create a table without a primary key in PostgreSQL. 🤯

Reply:
I was quite surprised as well